src/EventPayload.ts

```diff
@@ -71,22 +79,52 @@ export interface EventPayload {
   internalUser?: boolean;
   /** The IP address for the event.*/
   ip?: {
-    /** The IPv4 address associated with the event. */
+    /** The IPv4 address associated with the event.
+     *
+     * If not specified the system will use the Source IP address
+     * from the IP header of the request.
+     */
     address: string;
     /** The algorithm to use to anonymize the IP address after collection. */
     algorithm: string;
   };
   /** A label assigned to the event, e.g. a CTA label. */
   label?: string;
-  /** The locale of the user who generated the event. */
+  /** The locale of the user who generated the event.
+   *
+   * If not specified the system will automatically
+   * determine the locale from the Accept-Language header.
+   */
   locale?: string;
   /** The location information of the visitor for the event.
    * Either a Coordinates object with both latitude and longitude or a string
-   * with the country of the visitor for the event, as a ISO 3166-1 alpha-2 country code. */
+   * with the country of the visitor for the event, as a ISO 3166-1 alpha-2 country code.
```

Comment:
Would it be too much to provide a link to the set of ISO 3166-1 alpha-2 codes? Looking at the existing codes, most common countries are generally straightforward, but might be useful to provide a direct link to look it up. Wonder what your opinion on this is? Might need to bounce this off with Chris.